package.json

```diff
@@ -2,14 +2,15 @@
   "name": "peircemyheart",
   "version": "0.0.2",
   "description": "A web application for modeling Charles Peirce's Alpha Existential Graph System. In development for UAlbany 499.",
-  "main": "index.js",
+  "main": "build/index.html",
   "scripts": {
     "lint": "gts lint",
     "fix": "gts fix",
     "build": "vite build --mode production",
     "build-dev": "vite build --mode development",
-    "live-edit": "vite serve --open",
     "build-docs": "typedoc",
+    "build-electron": "vite build --mode electron && electron-packager build peircemyheart",
```

Comment:
Please add an overwrite script -
`"build-electron-ow": "vite build --mode electron && electron-packager build peircemyheart --overwrite"`

Reply:
What does this do? Vite will be default overwrite the build folder, is the folder with the electron app compiled by build not being overwritten?

Reply:
Some modules do get overwritten. From my testing, when I changed the window size on electron.js it did not get applied. So, I'm assuming core electron.js files don't get overwritten when doing regular build.

Reply:
Thanks, good to know.